iidm/iidm-api/src/main/java/com/powsybl/iidm/network/limitreduction/result/AbstractReducedLoadingLimits.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.network.limitreduction.result;
+
+import com.powsybl.iidm.network.LoadingLimits;
+
+import java.util.Collection;
+import java.util.Optional;
+import java.util.TreeMap;
+
+/**
+ * <p>Simple abstract implementation of {@link LoadingLimits} not linked to a network element, used to provide
+ * reduced limits without altering the real limits of the network element.</p>
+ * @author Olivier Perrin {@literal <olivier.perrin at rte-france.com>}
+ */
+public abstract class AbstractReducedLoadingLimits implements LoadingLimits {
+    private final double permanentLimit;
+    private final double originalPermanentLimit;
+    private final double reductionAppliedOnPermanentLimit;
+    private final TreeMap<Integer, TemporaryLimit> temporaryLimits = new TreeMap<>();
+
+    public record ReducedTemporaryLimit(String name, double value, int acceptableDuration, boolean fictitious,
+                                 double originalValue, double appliedReduction) implements TemporaryLimit {
+        @Override
+        public String getName() {
+            return name();
+        }
+
+        @Override
+        public double getValue() {
+            return value();
+        }
+
+        @Override
+        public int getAcceptableDuration() {
+            return acceptableDuration();
+        }
+
+        @Override
+        public boolean isFictitious() {
```

Comment:
isFictitious?

Reply:
This element is defined in `LoadingLimits.TemporaryLimit`. Its javadoc states:

> Check if the temporary limit is a real limit corresponding to an overloading  protection or just an operating rule